src/apis.php

```diff
@@ -418,6 +409,32 @@ protected function cron()
     }
 
     /**
+     * @param array $cronJobs
+     *
+     * @return bool
+     */
+    protected function containsOnlyAllowedCronJobs(array $cronJobs)
+    {
+        $validCronJobs = true;
+
+        if (empty($cronJobs)) {
+            return $validCronJobs;
+        }
+
+        foreach ($cronJobs as $cronJob) {
+            if (
+                empty($cronJob['job_name'])
```

Comment:
why is there an empty check?
The in_array check is sufficient. Is there a case I don't come up with?
I also think you should use the strict mode of in_array

Reply:
please check src/apis.php line 369 - it seems it happened in the past. Maybe we can restructure and throw the exception in the newly created method